### DISCUSSION TITLE
Problems in Using Data and GIS - Ethical Data Collection

### DISCUSSION THREAD

**Discussion by Wny-Duong** [albertkun/21S-ASIAAM-191A#29] [Past Thinking Caps]
I've noticed a lot of fellow students already discuss issues regarding how to properly disaggregate data and applying data towards ethical use cases, but I feel it's also very important to address issues with the data starting from the process of collecting it. There are many historical cases of data being collected unethically. More recently, however, there have been greater strides in trying to collect and use data in ethical fashions I've looked through a few papers and videos discussing the issues that I'll try to summarize here.

- (Crash Course video on Ethical Data Collection) https://www.youtube.com/watch?v=CzNANZnoiRs
This video discusses some historical cases of unethical data collection in the world. These instances included the study of the digestive system on an infirm man (Alexis St. Martin), the Tuskegee Syphillis Study (which was considered ethically unjustified since even the 1970s), and  most notably, the collection of HeLa cells used for biological research without compensation to the patient, Henrietta Lacks, or their family. What these cases all had in common, looking back, was that researchers lacked consideration for the people from whom their data was sourced. There were failures in informing subjects about the intent of their experiments, misinformation on how the studies would serve them directly, and even straight-up neglect and harm inflicted to study patients. 

-  (Unicef Article on Ethical Concerns with Geospatial Technology) https://www.unicef-irc.org/publications/pdf/DP%202018%2002.pdf
For anyone unfamiliar, UNICEF is a humanitarian agency associated with the United Nations that typically deals with issues of social welfare in developing nations. This paper discusses some of the policies and considerations they use in order to ensure that the GIS data they collect meets some degree of ethical standard. It includes suggestions such as making sure that the data collected addresses some policy or need of the community it's collected from, that only necessary data is collected, geomasking data to prevent identification of individuals in the study, and a lot more (p. 19-22).  Something I found very interesting in this paper specifically was the section it mentioned on the use of UAVs (drones) in collecting geographical data. Due to the stigma created from popular associations with drones, areas of conflict, and the US military, there is a lot of advice given on how to ensure that the data is collected properly and without distressing local communities. Given what we've been studying in the course so far and the professor's insistence that we try to avoid over-relying on secondary sources of data from larger government/research agencies, I feel that this paper would be very useful in instructing us on how to tailor our own data collection for our final projects later on.

- (Paper on ethical/legal issues regarding Asian/Pacific Islanders in medical study) https://www.ncbi.nlm.nih.gov/books/NBK236569/
This is a pretty long paper discussing issues such as the need to disaggregate the Asian demographic when discussing health and morbidity rates, the issue with relying on dated sources of data for analyzing healthcare of Asian-Americans, and the underdevelopment of databases to properly describe the Asian population and subsets in it even when there is a push to include Asian Americans and Pacific Islanders in more modern clinical studies. Whereas the previous two things I linked discussed how harm is done with unethical methods of data colection, this article emphasizes the issue of harm done due to a lack of study of minority AAPI subgroups and encourages more studies to be done and data to be collected. Both of these things however, must be done in a manner that is ethical, oriented towards solving specific problems, and of benefit to the AAPI communities. An issue more specific to the Asian American/Pacific Islander communities brought up here is the need and challenge in acquiring informed consent from people being studied, which is problematic due to issues of English fluency in the populations, illiteracy in certain older demographics, and the unfamiliarity of those people with ethically conducted studies (this more specifically leads to people not questioning the research they are subjected to and accepting studies despite not understanding the risks and details of those studies).

**Comment by avaitt** [comment]
Hi!

I love your take on this. When doing research I also found the Unicef Article on Ethical Concerns with Geospatial Technology and thought it was riveting. I had no idea GIS could instigate stigmas or discrimination based on region. It really made me think about how powerful yet frightening mapping software can be. When used for good, data visualizations (and data in general) can help address global problems, and pinpoint areas of crisis. However, if data is not used properly it can propagate harmful narratives that can cause distress to marginalized communities.

**Comment by jason-muljadi** [comment]
This is a really important post and your examples are incredibly valid for all those who are going into research and into studying. Oftentimes, ethics is thrown away for the sake of collecting data, which reflects societal norms. I especially am grateful that you mentioned Henrietta Lacks and the usage of HeLa cells that are still being used in this day for research.

Specifically for GIS, privacy tends to be negated for the sake of photo mapping out locations. I wonder about the mapping of Google Maps and assigning personal data to these geographical locations, which seems to be easier. The gamification of the deletion of personal privacy, as seen in games such as Geoguessr, may contribute to a world in which privacy erasure is normalized.
